demo/hc-nested-list.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>hc-nested-list demo</title>
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
+    <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+    <link rel="import" href="../hc-nested-list.html">
+    <link rel="import" href="../hc-nested-list-item.html">
+    <link rel="import" href="../test/utils.html">
+
+    <style is="custom-style" include="demo-pages-shared-styles">
+    </style>
+  </head>
+  <body>
+    <div class="vertical-section-container centered">
+      <h3><code>hc-nested-list demo</code></h3>
+      <demo-snippet>
+        <template>
+          <hc-nested-list-demo></hc-nested-list-demo>
+          <dom-module id="hc-nested-list-demo">
+            <template>
+              <style>
+                .item {
+                  cursor: pointer;
+                }
+                .item:hover, .item[open] {
+                  font-weight: bold;
+                }
+                [slot=loader] {
+                  text-align: center;
+                }
+                hc-nested-list {
+                  --hc-nested-list-loader: {
+                    background-color: rgba(0, 0, 0, 0.6);
+                  };
+                }
+              </style>
+
+              <hc-nested-list items="{{items}}">
+                <div slot="loader">
+                  test loader!!!
+                </div>
+                <template>
+                  <div class="item" open$="[[item.open]]">
+                    <hc-nested-list-item item="{{item}}">
+                      <span>[[item.label]]</span>
+                    </hc-nested-list-item>
+                  </div>
+                </template>
+              </hc-nested-list>
+            </template>
+            <script>
+              Polymer({
+                is: 'hc-nested-list-demo',
+                properties: {
+                  items: {
+                    type: Array,
+                    // This function is provided as part of test/utils.html, used for testing
+                    value: () => window.buildItems(100, 100, 100)
```

Comment:
That converts this to:
```
value() {
  return window.buildItems(100, 100, 100);
}
```
I think I prefer the arrow function to remove total number of characters.  In the cases where I'm already required to write a return it makes sense though.